google/ggrc-core
# GGRC-4821 Object Review - Review is not applied for objects that already have a reviewer

# Issue description

Can't approve object review. 

# Steps to test the changes


1) create control 
2) create workflow 
3) create task group 
4) create task 
5) map that object to workflow 
6) activate workflow 
7) go to control info_widget (you shouldn't see that review is started)
8) create object review 
9) approve review 

# Solution description

CTGOT instances was not filtered by field `object_approval`. That's why FE send on verify procedure first object from CTGOT list. And sometimes that instance isn't related to review procedure.    

# Sanity checklist




# PR Review checklist
